README.md

```diff
@@ -148,31 +148,17 @@ Download the latest RPM package from [releases](https://github.com/yuezk/GlobalP
 ```bash
 sudo rpm -i globalprotect-openconnect-*.rpm
 ```
-### Gentoo
-
-Install from the ```rios``` or ```slonko``` overlays.  Example using rios:
 
-#### 1. Enable the overlay
-```
-sudo eselect repository enable rios
-```
+### Gentoo
 
-#### 2. Sync with the repository
+It is available via `guru` and `lamdness` overlays.
 
-  - If you have eix installed, use it:
-```
-sudo eix-sync
-```
-  - Otherwise, use:
-```
-sudo emerge --sync
+```bash
+sudo eselect repository enable guru
+sudo emerge -r guru sync
+sudo emerge -av net-vpn/globalprotect-openconnect
```

Comment:
I prefer using the `guru` overlay for this instruction since it is more official (I'm using the package from my `lameness` overlay though).